prusti-tests/tests/verify/pass/equality/quantifiers.rs

```diff
@@ -0,0 +1,12 @@
+use prusti_contracts::*;
+
+#[derive(PartialEq, Eq)]
+struct A { x: usize }
+
+#[requires(forall(|other: A| _foo == other))]
+fn test(_foo: A) {}
+
+#[requires(forall(|other: u32| _foo == other))]
```

Comment:
For `verify/pass`, this is quite a strange test case. As a user I would expect this to fail.

Reply:
It seems that Github got confused. This test is not part of the PR.